src/scripts/delete_cluster.sh

```diff
@@ -1,41 +1,41 @@
 #!/bin/bash
-ORB_EVAL_CLUSTER_NAME=$(circleci env subst "$ORB_EVAL_CLUSTER_NAME")
-ORB_EVAL_CONFIG_FILE=$(circleci env subst "$ORB_EVAL_CONFIG_FILE")
-ORB_EVAL_AWS_REGION=$(circleci env subst "$ORB_EVAL_AWS_REGION")
-ORB_EVAL_AWS_PROFILE=$(circleci env subst "$ORB_EVAL_AWS_PROFILE")
-ORB_EVAL_CFN_ROLE_ARN=$(circleci env subst "$ORB_EVAL_CFN_ROLE_ARN")
-ORB_EVAL_AWS_MAX_POLLING_WAIT_TIME=$(circleci env subst "$ORB_EVAL_AWS_MAX_POLLING_WAIT_TIME")
+AWS_EKS_STR_CLUSTER_NAME=$(circleci env subst "$AWS_EKS_STR_CLUSTER_NAME")
+AWS_EKS_STR_CONFIG_FILE=$(circleci env subst "$AWS_EKS_STR_CONFIG_FILE")
+AWS_EKS_STR_REGION=$(circleci env subst "$AWS_EKS_STR_REGION")
+AWS_EKS_STR_PROFILE=$(circleci env subst "$AWS_EKS_STR_PROFILE")
+AWS_EKS_STR_CFN_ROLE_ARN=$(circleci env subst "$AWS_EKS_STR_CFN_ROLE_ARN")
+AWS_EKS_STR_MAX_POLLING_WAIT_TIME=$(circleci env subst "$AWS_EKS_STR_MAX_POLLING_WAIT_TIME")
```

Comment:
Per standard, shouldn't these follow the format below?

```
VALUE=$(echo "${[ORB]_STR_[NAME]}"| circleci env subst)
```

Reply:
done!